submissions/emmanuelEsuong/introductionToHtml/index.html

```diff
@@ -0,0 +1,214 @@
+<<!DOCTYPE html>
+<html lang="en">
+<head>
+    <p> <img src="Image/Foodco upmata.PNG" controls></img></p>
+    
+    <meta charset="UTF-8">
+    <title>Welcome To Foodco Online Store</title>
+<!--Foodco Head logo-->
+        <p style="text-align:center;"><img src="Image/Foodco%20logo%20one.PNG" alt="centred image"></img></p>
+            <table style="border-spacing:20px;margin-left:auto;margin-right:auto;">
+                <tr>
+                    <td><a href="#"<b>HOME</b></a></td>
+
+                    <td><a href="#"<b>ABOUT US</b></a></td>
+
+                    <td><a href="#"<b>BUSINESS UNITS</b></a></td>
+
+                    <td><a href="#"<b>FOODCO FOUNDATION</b></a></td>
+
+                    <td><a href="#"<b>CAREERS</b></a></td>
+
+                    <td><a href="#"<b>LOCATIONS</b></a></td>
+
+                    <td><a href="#"<b>WHAT'S NEW</b></a></td>
+
+                    <td><a href="#"<b>CONTACT US</b></a></td>
+
+                    <td><a href="#"<b>PROMOS</b></a></td>
+                <tr>
+            </table>
+        </p>
+</head>
+
+<body>
+    <!--Products ads animation-->
+    <p style="text-align:center;"><img src="Image/Foodco%20ads.gif" width="1550" height="600" controls></img> </p>
+    
+    <!--Menu list-->    
+    <table style="border-spacing:20px;margin-left:auto;margin-right:auto;">
+        <tr>
+            <td> <img src="Image/Foodco s.PNG" controls></img> </td>
```

Comment:
The images used in the file are not added to this pull request. Update that.

Reply:
You are meant to use only 2 product images. Check the criteria here.
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7